File: ui/vzk-ui/src/app/models/game/game.ts
export class Game {
  id: string;
  title: string;
  img: string;
  logo: string;

  constructor(id: string, title: string, img: string, logo: string) {
    this.id = id;
    this.title = title;
    this.img = img;
    this.logo = logo;
  }
}

File: account/src/main/java/com/vzk/account/models/Game.java
package com.vzk.account.models;

import jakarta.persistence.Basic;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.*;

import java.util.UUID;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity(name = "game")
public class Game {
    @Id
    @Column(name = "id")
    private UUID id;

    @Basic
    @Column(name = "title")
    private String title;

    @Basic
    @Column(name = "img")
    private String image;

    @Basic
    @Column(name = "logo")
    private String logo;
}